charts/ota-core/values.yaml

```diff
@@ -0,0 +1,53 @@
+replicaCount: 1
+
+image:
+  repository: advancedtelematic/sota-core
```

Comment:
what is this thing? I can't find source code for it

Reply:
@doanac  is correct, it is not part of ce and should be removed

Reply:
> @doanac is correct, it is not part of ce and should be removed

Done.